com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/UV/PolarCoordinatesNode.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class PolarCoordinatesNode : IStandardNode
```

Comment:
Got a shader error on instantiating this node: 
![image](https://user-images.githubusercontent.com/13033542/167045042-44c44369-72e5-4de2-a9b3-1b45f419300a.png)

Is this a known issue?

Reply:
Fixed